playwright/_impl/_locator.py

```diff
@@ -528,7 +527,7 @@ async def screenshot(
         params = locals_to_params(locals())
         return await self._with_element(
             lambda h, timeout: h.screenshot(
-                **ChainMap({"timeout": timeout}, params),
```

Comment:
This reverts https://github.com/microsoft/playwright-python/pull/1877 which seems not relevant anymore since our linters are happy.